wgengine/router/dns/resolved.go

```diff
@@ -55,8 +47,8 @@ type resolvedLinkDomain struct {
 	RoutingOnly bool
 }
 
-// resolvedIsActive determines if resolved is currently managing system DNS settings.
-func resolvedIsActive() bool {
+// isResolvedActive determines if resolved is currently managing system DNS settings.
+func isResolvedActive() bool {
```

Comment:
Best left for some future work, but a question: this appears to only be checked on startup. But if we are started by systemd and the config file does not contain a direct dependency on the network service being running, isn't there a good chance this will be called before resolved is running? Wouldn't it be better to delay this check until the first time DNS is configured (which is after controlclient talks to a server, and so we expect the network to be running)?